remoteit/desktop
# DESK-1003 on demand connections

### Description


Show "Create connection" instead of connect as primary button so that it indicates we're not connecting yet and add timeout setting.

- Allow spaces in path names for mac
- Support for non-commercial and commercial licenses
- No more light icons
- Routing fixes
- Fix for map undefined error
- Avatar on settings page for small screen
- Double panel sizing fix

### Test plan



1. Sign in
2. Go to service 
3. Click create connection
4. Launch connection
5. It should connect when the request is send, not the button clicked

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1003>
